tools/build_pytorch_libs.py

```diff
@@ -181,7 +181,15 @@ def run_cmake(version,
         USE_REDIS=os.getenv('USE_REDIS'),
         USE_GLOG=os.getenv('USE_GLOG'),
         USE_GFLAGS=os.getenv('USE_GFLAGS'),
-        WERROR=os.getenv('WERROR'))
+        WERROR=os.getenv('WERROR'),
+        # openmp cmake flag
+        # If we setup MACOSX_DEPLOYMENT_TARGET in mac, we should also set the
+        # libomp library flags explicitly.
+        OpenMP_C_FLAGS=os.getenv('OpenMP_C_FLAGS'),
```

Comment:
Are these really needed? Can't you instead use general purpose flags like `CFLAGS` and `LDFLAGS`?

Reply:
The change in build_pytorch_libs.py are removed now.